pkg/plugin/aggregation/status.go

```diff
@@ -82,9 +82,12 @@ func GetStatus(client kubernetes.Interface, namespace string) (*Status, error) {
 		return nil, errors.Wrap(err, "sonobuoy namespace does not exist")
 	}
 
+	// Determine sonobuoy pod name
+	SetStatusPodName(client, namespace)
+
 	pod, err := client.CoreV1().Pods(namespace).Get(StatusPodName, metav1.GetOptions{})
 	if err != nil {
-		return nil, errors.Wrap(err, "could not retrieve sonobuoy pod")
+		return nil, errors.New("could not retrieve sonobuoy pod")
```

Comment:
Why the change in the error? Seems like that would just obfuscate issues by not returning the actual error to the user.

Reply:
To test this error I created a second pod with the 'run:sonobuoy-master' label in the same namespace.

This is the error if we don't make this change:
```
$ sonobuoy status
ERRO[0001] More than one pod with label 'run:sonobuoy-master'
ERRO[0001] error attempting to run sonobuoy: could not retrieve sonobuoy pod: pods "sonobuoy" not found
```

This seemed a little confusing to me b/c the user will wonder "why are you looking for a pod called 'sonobuoy' - my pod has a different name?"

This happens because the `SetStatusPodName` function doesn't set the pod name if there's an error and it reverts back to the default name "sonobuoy".  (I'm wondering now if this was wise.)

This is the error output if we *do* make this change:
```
$ sonobuoy status
ERRO[0006] More than one pod with label 'run:sonobuoy-master'
ERRO[0006] error attempting to run sonobuoy: could not retrieve sonobuoy pod
```
With the both errors provided together, I think this is clearer.  But open to suggestions.

Reply:
Update for clarity: Due to other changes based on other review items...

There is no longer an error if there are multiple pods with the `run:sonobuoy-master` label.  The first pod is selected (with a warning) and the status command runs.

If there are *no* pods with the label the output is now as follows:
```
$ sonobuoy status
ERRO[0001] No pods found with label 'run=sonobuoy-master' in namespace heptio-sonobuoy, using default pod name 'sonobuoy'
ERRO[0001] error attempting to run sonobuoy: could not retrieve sonobuoy pod
```